src/lambda/manageUserStatus.js

```diff
@@ -0,0 +1,72 @@
+// handles request to approve/ reject user awaiting verification or super user status
+
+import connectToDatabase from './database/dbConnection';
+import {
+  approveSuperUser,
+  rejectSuperUser,
+  approveUser,
+  rejectUser,
+  getUser,
+} from './database/queries/user';
+import { status } from '../constants/users';
+
+export async function handler(event, context) {
+  // eslint-disable-next-line no-param-reassign
+  context.callbackWaitsForEmptyEventLoop = false;
+
+  try {
+    await connectToDatabase();
+    const { user, admin, action, userStatus } = JSON.parse(event.body);
```

Comment:
I think you should validate the data here. check if it exists and it's what's expected

Reply:
good point I'll do that